### PR TITLE
[SDK][INCLUDE] Add WM_WTSSESSION_CHANGE message

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: N/A

## Proposed changes
- Add `WM_WTSSESSION_CHANGE` messages to `<winuser.h>`.
- Add `WTS_*` constant macros.

### CODE REVIEW

**Comment by katahiromz** [comment]
For your information:
https://docs.microsoft.com/en-us/windows/win32/termserv/wm-wtssession-change

File: sdk/include/psdk/winuser.h [hunk 1]
```diff
@@ -1825,6 +1825,19 @@ extern "C" {
...
+#define WTS_CONSOLE_DISCONNECT 0x2
+#define WTS_REMOTE_CONNECT 0x3
+#define WTS_REMOTE_DISCONNECT 0x4
+#define WTS_SESSION_LOGON 0x5
+#define WTS_SESSION_LOGOFF 0x6
+#define WTS_SESSION_LOCK 0x7
+#define WTS_SESSION_UNLOCK 0x8
+#define WTS_SESSION_REMOTE_CONTROL 0x9
+#define WTS_SESSION_CREATE 0xa
+#define WTS_SESSION_TERMINATE 0xb
```
Comment (binarymaster) [inline sdk/include/psdk/winuser.h]: Nitpick: 0xA and 0xB in uppercase, for consistence.

Comment (katahiromz) [reply sdk/include/psdk/winuser.h]: OK.

File: sdk/include/psdk/winuser.h [hunk 2]
```diff
@@ -1825,6 +1825,19 @@ extern "C" {
 #define WM_MOUSEHOVER               0x02A1
 #define WM_MOUSELEAVE               0x02A3
 
+#define WTS_CONSOLE_CONNECT 0x1
```
Comment (HBelusca) [inline sdk/include/psdk/winuser.h]: We have to check whether this is Vista+; if so, this has to be surrounded by suitable #ifdefs.

Comment (katahiromz) [reply sdk/include/psdk/winuser.h]: `#if (WINVER >= 0x0600)` added.

Comment (HBelusca) [reply sdk/include/psdk/winuser.h]: To be compatible with MS it would be `#if(_WIN32_WINNT >= 0x0501)`

**Pull request merged**